Release notes

biodranik opened the pull request:
Please help us with translations to your language. Just comment with a proper version. And don't forget about the max 500 characters limit for Android.

@rtsisyk настрой, пожалуйста, чтобы при обновлении вот этих файлов, автоматически назначались ревьюеры на этих языках. И для ведра и для айоса.

rtsisyk commented on android/src/fdroid/play/release-notes/ru-RU/default.txt:
```diff
@@ -1,13 +1,12 @@
-• Обновленные переводы
+• Свежие карты OSM за 19 ноября
+• Исправлена скорость велосипеда для маршрутизации
+• Mastodon в "?"
```
Непонятно

rtsisyk commented on iphone/metadata/en-US/release_notes.txt:
```diff
@@ -1,38 +1,24 @@
+• Fixed opening some URLs with # and ? symbols
+• Fixed opening Matrix link from About
+• Added Mastodon in About
+• Removed ugly line in menu
```
Too generic and doesn't bring any value for user.

Ferenc- commented:
Can the update to `qt/res/app.organicmaps.desktop.metainfo.xml` also be part of this PR or does it have to be different?

siiky commented:
Some rough-ish pt-PT:

```
• Novos dados OSM desde 19 de Novembro
• Corrigir velocidade de bicicleta em navegação
• Adicionar Mastodon em "?"
• Corrigir distância em resultados de procura
• Corrigir ícones sobrepostos
• Adicionar highway=busway e fisioterapeuta
• Ilhas e arquipélagos no Mapa Mundo
• Corrigir prioridades de renderização de elementos de água e waterway=dam
• Corrigir áreas de lazer
• Adicionar traduções de et, actualizar cs, es, nb, tr
• Corrigir categorias RV e catalã
```

Skipped "Fixed some crashes", "Metro ...", and "navigation menu under navbar" to stay inside the 500 limit.

matheusgomesms reviewed (changes requested):
PT-BR:

Android

• Novos dados OSM de 19/11
• Velocidade de bicicleta para melhor roteamento
• Adição de Mastodon em "?"
• Correção de alguns erros e do menu de navegação
• Correção de distância na busca
• Correção de ícones sobrepostos e de áreas de lazer
• Adição de BRTs e fisioterapeuta
• Ilhas e arquipélagos no mapa mundial
• Correção de prioridade de exibição de água/píer e barragem
• Tradução e atualização de cs, es, et, nb, tr
• Metrô para en-GB
• Correção de categorias de RV e catalão
…mais: omaps.app/news
iOS

• Novos dados OSM de 19 de novembro
• Correção de velocidade de bicicleta para melhor roteamento
• Correção de abertura de algumas URLs com os símbolos # e ?
• Correção da abertura do link Matrix da seção de Ajuda
• Adição de Mastodon na seção de Ajuda
• Mostrar nome na língua local ao selecionar objeto
• Correção ao mostrar a hora de fechamento de um local quando faltava exatamente 1 hora para o encerramento

Estilos
 • Correção de alguns objetos que se sobrepunham
 • Adição de BRTs e fisioterapeuta
 • Ilhas e arquipélagos são agora visíveis no mapa mundial
 • Exibir barragem como área de fundo
 • Prioridade da exibição de água menor do que píer
 • Correção de áreas de lazer

Localização
 • Adição de tradução em Estoniano
 • Atualização das traduções em checo, espanhol, norueguês e turco
 • Links do site principal agora estão localizados no app, favor nos ajudar com a tradução: https://github.com/organicmaps/organicmaps.github.io
 • Metrô para inglês Britânico
 • Correção de tradução ausente para categoria autocaravana
 • Correção de busca de categoria para catalão

biodranik commented:
> Can the update to qt/res/app.organicmaps.desktop.metainfo.xml also be part of this PR or does it have to be different?

@Ferenc- yes, here is the list of changes:

New data from 221119
Corrected bicycle's speed for better routing
Fixed map dragging on Linux
Use "+" key to zoom in
Improve docs about building OM with WSL
Use high-resolution icons on high-DPI displays
Fixed some improperly overlapping types
Added highway=busway and physiotherapist
Islands and archipelagos are now visible on World Map
Draw waterway=dam as background area
Water draw priority is lesser now than pier
Fixed leisure areas

MetehanOzyurek commented:
Android:
• Yeni OSM harita verileri (19 Kasım)
• Rotalar için bisiklet hızı düzeltildi
• "?" bölümüne Mastodon sayfamız eklendi
• Bazı hatalar düzeltildi
• Navigasyon menüsü düzeltildi
• Artık arama sonuçlarında mesafe gösteriliyor
• Üst üste binen bazı simgeler düzeltildi
• highway=busway yolu ve fizyoterapist eklendi
• Dünya haritasına adalar ve takımadalar eklendi
• İskele, bent ve iskele üzerindeki su çizimi düzeltildi
…dahası: omaps.app/news

MetehanOzyurek commented:
App Store:
• Yeni OSM harita verileri (19 Kasım)
• Daha iyi rotalar için hesaplama için kullanılan bisiklet hızı düzeltildi
• Bazı URL'lerin # ve ? sembolleri ile açılması sorunu düzeltildi
• Hakkında kısmından Matrix bağlantısının açılmaması sorunu düzeltildi
• Hakkında bölümüne Mastodon sayfamız eklendi
• Mekan Sayfasında yerin adı artık yerel dilde gösteriliyor
• Yerin kapanmasına tam olarak 1 saat kaldığında daha iyi şekilde gösteriliyor

Stiller
• Bazı yanlış örtüşen tipler düzeltildi
• highway=busway (otobüs yolu) ve fizyoterapist kategorisi eklendi
• Adalar ve takımadalar artık dünya haritasında görülebilir
• waterway=dam artık arka plan alanı olarak çiziliyor
• Su kütleleri artık iskelelerin üstüne çizilmiyor
• Eğlence alanları düzeltildi

Çeviriler
• Çeviriler güncellendi / düzeltildi.
• Uygulama içerisinde web sitemize verilen bağlantılar artık Türkçe sayfalara yönlendiriyor.
• Eksik karavan kategorileri çevirileri düzeltildi

d4f5409d commented:
F-Droid hu:
• Új OSM adatok november 19-től
• A kerékpársebesség hibás megjelenítésének a javítása
• Mastodon hozzáadása a "?"-hez
• Néhány összeomlás javítása
• Navigációs menü javítása a navigációs sáv alatt
• A keresési eredményekben már megjelenik a távolság
• Néhány egymást fedő ikon javítása
• highway=busway és fizioterapeuta hozzáadása
• Szigetek és szigetcsoportok a világtérképen
• water/peer és waterway=dam rajzolási prioritás javítása
• leisure területek javítása
• Észt nyelv hozzáadása, cs, es, nb, tr fordítás frissítése
• Metro az en-GB-hez
• RV és katalán kategóriák javítása
...több: omaps.app/news
